pyaedt/sbrplus/hdm_parser.py

```diff
@@ -42,11 +42,11 @@ def __init__(self, filename):
         self.binarycontent = binarycontent
 
     def parse_message(self):
-        """Parses the binary content of the HDM file"""
+        """Parse the binary content of the HDM file."""
         return self._parse(self.message["type"])
 
     def _parse(self, type_name):
-        """Generic parser method, dispatches to specialized ones"""
+        """Generic parser method, dispatches to specialized ones."""
```

Comment:
Method description should start with a verb. Would we be able to use this instead: "Use generic parser method, which dispatches to specialized parsers."? What are we parsing?